### ISSUE TITLE
Trwam TV

### ISSUE THREAD

**Issue by kamilinho1** [ramicspa/kodi#558]
http://tv-trwam.pl/na-zywo
Dasz radę stworzyć wtyczkę do tego źródła?
Z góry dziękuję😀

**Comment by tomasiek** [comment]
Zanim odezwą się głosy różnych opcji... mamy tvn24, mamy polsat news... może coś z innego źródła też będzie warto coś dodać? Wbrew pozorom i powszechnie panującym opiniom są tam audycje naprawdę wartościowe....

**Comment by kamilinho1** [comment]
I darmowe😀

**Comment by bart00777** [comment]
Na wp pilot masz ten kanał za free ;-)

**Comment by kamilinho1** [comment]
Tak, doskonale o tym wiem. Dodanie bezpośredniego streamu pozwoli na oglądanie kanału nawet gdy wtyczka wp.pilot przestaje działać. Co ma miejsce na przykład dzisiaj. Stream ze strony działa bez dodatkowych logowań itp.

**Comment by ramicspa** [comment]
ja tutaj podziekuje ... wystarczy ze wp pilot nadaje.

**Comment by gregor35** [comment]
@kamilinho1 to zwykły prowokator niech idzie na onet.pl.

**Comment by tomasiek** [comment]
@gregor35 dlaczego prowokator? 
@kamilinho1 jak ramic nie da rady... to może ja w końcu spróbuję... dam znać tutaj.

**Comment by kamilinho1** [comment]
Dzięki, powodzenia życzę. 

20.12.2017 napisał:
>@gregor35 dlaczego prowokator? 
>@kamilinho1 jak ramic nie da rady... to może ja w końcu spróbuję... dam
>znać tutaj.
